input/pagecontent/data.md

```diff
@@ -1927,6 +1927,47 @@ This Rule Type is not printed.
    </tbody>
 </table>
 
+##### Safety
+
+Not currently used
```

Comment:
Looks good - but this comment could age as we don't know when or if they will bring it back